documentation/diff-reports-previous-version/AgentOntology.md

```diff
@@ -0,0 +1,188 @@
+
+#### Added
+- [is subordinate role to](http://www.ontologyrepository.com/CommonCoreOntologies/is_subordinate_role_to) [definition](http://www.ontologyrepository.com/CommonCoreOntologies/definition) "For all x,y,t: y is subordinate role to x at t iff: x is an instance of Role at time t, and y is an instance of Role at time t, and there is some z such that x is realized by z and z is an instance of Process which creates, modifies, transfers, or eliminates some u such that u is a Process Regulation at time t, and u is addressed to the bearer of y."@en 
+
+
+### permits `http://www.ontologyrepository.com/CommonCoreOntologies/permits`
+#### Removed
+- [permits](http://www.ontologyrepository.com/CommonCoreOntologies/permits) [definition](http://www.ontologyrepository.com/CommonCoreOntologies/definition) "x permits y at t iff: x is an instance of Action Regulation at time t, and y is an instance of Act at time t, and x prescribes that some agent may be agent in y."@en 
+
+#### Added
+- [permits](http://www.ontologyrepository.com/CommonCoreOntologies/permits) [definition](http://www.ontologyrepository.com/CommonCoreOntologies/definition) "x permits y at t iff: x is an instance of Process Regulation at time t, and y is an instance of Process at time t, and x prescribes that y may occur."@en 
+
+
+### prohibits `http://www.ontologyrepository.com/CommonCoreOntologies/prohibits`
+#### Removed
+- [prohibits](http://www.ontologyrepository.com/CommonCoreOntologies/prohibits) [definition](http://www.ontologyrepository.com/CommonCoreOntologies/definition) "x prohibits y at t iff: x is an instance of Action Regulation at time t, and y is an instance of Act at time t, and x prescribes that some agent must not be agent in y."@en 
+
+#### Added
+- [prohibits](http://www.ontologyrepository.com/CommonCoreOntologies/prohibits) [definition](http://www.ontologyrepository.com/CommonCoreOntologies/definition) "x prohibits y at t iff: x is an instance of Process Regulation at time t, and y is an instance of Process at time t, and x prescribes that some that y must not occur."@en 
+
+
+### requires `http://www.ontologyrepository.com/CommonCoreOntologies/requires`
+#### Removed
+- [requires](http://www.ontologyrepository.com/CommonCoreOntologies/requires) [definition](http://www.ontologyrepository.com/CommonCoreOntologies/definition) "x requires y at t iff: x is an instance of Action Regulation at time t, and y is an instance of Act at time t, and x prescribes that some agent must be agent in y."@en 
+
+#### Added
+- [requires](http://www.ontologyrepository.com/CommonCoreOntologies/requires) [definition](http://www.ontologyrepository.com/CommonCoreOntologies/definition) "x requires y at t iff: x is an instance of Process Regulation at time t, and y is an instance of Process at time t, and x prescribes that some that y must occur."@en 
```

Comment:
I believe "x prescribes that some that y must occur" should be "x prescribes that some y must occur"